### PR TITLE
Adapt variables for image naming according to upstream PR#740

### PR DESCRIPTION
**Pull request by thehejik**
### What does this PR do?
Fixes part building turtles chart and image

### Checklist:
- [x] Squashed commits into logical changes
- [ ] Documentation
- [x] GitHub Actions (if applicable) https://github.com/rancher/rancher-turtles-e2e/actions/runs/10903819762/job/30258817260

### CODE REVIEW

File: .github/workflows/master-e2e.yaml [hunk 1]
```diff
@@ -220,7 +220,7 @@ jobs:
       QASE_REPORT: 1
       TAG: 0.0.1
       TURTLES_REPO: rancher-sandbox/rancher-turtles
```
Comment (cpinjani) [inline .github/workflows/master-e2e.yaml:222]:
```suggestion
      TURTLES_REPO: rancher/turtles
```

**Review by cpinjani — APPROVED** [review]
looks good!

File: .github/workflows/master-e2e.yaml (hunk 1 above)
Comment (cpinjani) [reply .github/workflows/master-e2e.yaml:222]: Please see if we can include this change.

Comment (thehejik) [reply .github/workflows/master-e2e.yaml]: Thanks, let see https://github.com/rancher/rancher-turtles-e2e/actions/runs/10904964024

Comment (thehejik) [reply .github/workflows/master-e2e.yaml]: Seems it works.

**Pull request merged**